### PR TITLE
chore: update fedcm links

### PR DESCRIPTION
**Pull request by Delta456**
This PR updates the new FedCM links with the old ones.

https://w3c-fedid.github.io/FedCM/ is the only working link and the other link https://fedidcg.github.io/FedCM/ gives a 404 

image

### CODE REVIEW

File: explorations/alternatives_considered.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Alternatives Considered
 
-Now that we have a deep understanding of (a) the [problem](README.md) and (b) the [motivations](https://fedidcg.github.io/FedCM/#privacy-threat-model) and [topology](activation.md) of the parties involved, lets look at some **why not**s.
+Now that we have a deep understanding of (a) the [problem](README.md) and (b) the [motivations](https://w3cping.github.io/privacy-threat-model/) and [topology](activation.md) of the parties involved, lets look at some **why not**s.
```
Comment (samuelgoto) [inline explorations/alternatives_considered.md:3]:
```suggestion
Now that we have a deep understanding of (a) the [problem](README.md) and (b) the [motivations](https://w3c-fedid.github.io/FedCM/#privacy) and [topology](activation.md) of the parties involved, lets look at some **why not**s.
```

File: explorations/cookies.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Cookies
 
-This is a **proposal** for a high level API to support identity federation under this [threat model](https://fedidcg.github.io/FedCM/#privacy-threat-model).
+This is a **proposal** for a high level API to support identity federation under this [threat model](https://w3cping.github.io/privacy-threat-model/).
```
Comment (samuelgoto) [inline explorations/cookies.md:3]:
```suggestion
This is a **proposal** for a high level API to support identity federation under this [threat model](https://w3c-fedid.github.io/FedCM/#privacy).
```

File: explorations/cookies.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ This is a proposal to preserve these operations in the absence of third party co
...
+From a [privacy threat model](https://w3cping.github.io/privacy-threat-model/) perspective, the design of this proposal is anchored on the observation that the most critical moment is when the identities between the RP and the IDP are joined for the very first time, namely when the user creates a new account in the RP using the identifiers from the IDP or when a user signs-in to an RP with an IDP for the first time in the browser: once the identities are joined, any arbitrary/uncontrolled cross-side communication can happen (with or without the browser's permission, e.g. via backchannel or cookie-less requests).
```
Comment (samuelgoto) [inline explorations/cookies.md:21]: same here

Comment (Delta456) [reply explorations/cookies.md]: Done

**Review by wseltzer — APPROVED** [review]
LGTM

**Comment by samuelgoto** [comment]
Merging, since these are non substantive changes.

**Pull request merged**